lib/v2/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -29,60 +29,50 @@ class ExploreScreen extends StatelessWidget {
         appBar: AppBar(title: Text('Explore'.i18n)),
         body: BlocConsumer<ExploreBloc, ExploreState>(
           listenWhen: (_, current) => current.pageCommand != null,
-          listener: (context, state) async {
+          listener: (context, state) {
             var pageCommand = state.pageCommand;
             BlocProvider.of<ExploreBloc>(context)..add(const ClearExplorePageCommand());
             if (pageCommand is NavigateToRoute) {
-              bool? shouldReloadExplore = await NavigationService.of(context).navigateTo(pageCommand.route);
-              if (shouldReloadExplore != null && shouldReloadExplore) {
-                BlocProvider.of<ExploreBloc>(context)..add(const LoadExploreData());
-              }
+              NavigationService.of(context).navigateTo(pageCommand.route);
             }
           },
           builder: (context, ExploreState state) {
             switch (state.pageState) {
-              case PageState.initial:
-                return const SizedBox.shrink();
-              case PageState.loading:
-                return const FullPageLoadingIndicator();
               case PageState.failure:
                 return const FullPageErrorIndicator();
```

Comment:
I dont think we need fullPageErrorindicator() on this page anymore.

Reply:
But what if the is DHO member call fails?

Reply:
If it does, lets do a toast fail message

Reply:
we dont want to block the entire page, for this fail

Reply:
Also, lets move the loading indication to the widget thats loading, and make sure the other static widgets are always available

Reply:
I Add the local loading and also the snack for error. 

However, I do not know if that loading is necessary. Users who are not DHO members will always wonder what does that loding mean? I think same for the error.